Resources/ConfigPresets/SimpleStation14/Parkstation.toml

```diff
@@ -0,0 +1,142 @@
+[game]
+hostname = "[NA East][MRP] Parkstation"
+desc = "A server focused on RP expectations and calmer rounds."
+maxplayers = 64
+soft_max_players = 48
+lobbyenabled = true
+lobbyduration = 180
+role_timers = true
+fallbackpreset = "extended"
+map_pool = "ParkMapPool"
+
+[npc]
+enabled = true
+pathfinding = true
+
+[server]
+rules_file = "Park_Rules.txt"
+id = "parkstation"
+
+[whitelist]
+min_players = 32
+
+[rules]
+rules.time = 60
+
+[events]
+ramping_average_end_time = 120.0
+ramping_average_chaos = 7.0
+
+[traitor]
+min_players = 2
+players_per_traitor = 8
+starting_balance = 16
+
+[zombie]
+min_players = 9
+players_per_infected = 5
+
+
+[mapping]
+autosave_interval = 180.0
+
+[atmos]
+space_wind = true
+space_wind_max_velocity = 22.0
+
+[shuttle]
+emergency_early_launch_allowed = true
+emergency_dock_time = 260.0
+emergency_transit_time = 180.0
+recall_turning_point = 0.75
+auto_call_time = 150
+
+[biomass]
+easy_mode = false
+
+[ambience]
+restart_sounds_enabled = true
+
+[explosion]
+tiles_per_tick = 128
+incremental_tile = false
+
+[ic]
+restricted_names = false
+flavor_text = true
+
+[ooc]
+enable_during_round = true
+
+[vote]
+map_enabled = true
+restart_enabled = true
+restart_required_ratio = 0.75
+restart_not_allowed_when_admin_online = true
+timermap = 45
+
+
+[console]
+loginlocal = true
+
+[hub]
+advertise = true
+# server_url = "ss14://0.0.0.0:1212"
+
+[auth]
+# 0 = Optional, 1 = Required, 2 = Disabled
+mode = 1
+allowlocal = true
+
+[log]
+path = "logs"
+format = "log_%(date)s-%(time)s.txt"
+level = 1
+enabled = true
+
+[net]
+tickrate = 30
+port = 1212
+bindto = "::,0.0.0.0"
+log_late_msg = false
+
+[status]
+enabled = true
+
+
+[infolinks]
+discord = "https://discord.gg/49KeKwXc8g"
+github = "https://github.com/Park-Station/Parkstation"
+patreon = "https://www.patreon.com/simple14"
+
+[discord]
+# ahelp_webhook = "https://discord.com/api/webhooks/number/gibberish"
+# round_webhook = "https://discord.com/api/webhooks/number/gibberish"
+# round_roleid = "1091482392964890646"
```

Comment:
These probably shouldn't be _commented out_, just for clarity's sake. 
Ditto for all the other commented out info.